index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<title>PoGo GBL Lead Tracker</title>
+
+<body>
+  <div class=doctitle>Pokémon Go Battle League Lead Tracker - Documentation</div>
+  <h1>Jump To:</h1>
+  <p><a href="#great-league">Great League</a></p>
+  <p><a href="#ultra-league">Ultra League</a></p>
+  <p><a href="#master-league">Master League</a></p>
+  <div></div>
+  <h1><span class=get>GET</span>/leagues</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=get>GET</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues
+    <span class=desc>Headers</span>: Authorization: Bearer YOUR_API_KEY_HERE
+    <span class=desc>Description</span>: Returns a list of all Pokémon Go Battle leagues.
+  </code>
+  <h1 id="great-league"><span class=get>GET</span>/leagues/great-league/leads</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=get>GET</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues/great-league/leads
+    <span class=desc>Headers</span>: Authorization: Bearer YOUR_API_KEY_HERE
+    <span class=desc>Description</span>: Returns a list of all lead Pokémon players have encountered in Great League.
+  </code>
+  <h1><span class=get>GET</span>/leagues/great-league/leads/{slug}</h1>
+  <code>
+    <span class=desc>Method</span>: <span class=get>GET</span>
+    <span class=desc>Route</span>: https://placeholder.url/leagues/great-league/leads
```

Comment:
You could consider `https://placeholder.url/leagues/:league/leads` which would allow you to capture the name they provided and would simplify your code. Otherwise I suspect you'll end up with a lot of repetitive controllers.